docs/help-center/docs/warden-app/trade.md

```
@@ -0,0 +1,219 @@
+In the [AI Trading Terminal](#access-the-trading-terminal), you can access various market analysis tools helping you make informed decisions on your trades:
+
+- **The live chart** displays perp price movements. It supports multiple chart types, technical indicators, drawing tools, and more.
+- **The order book** shows real-time perp market liquidity. The market data for the chart and order book is provided by [Hyperliquid](https://hyperfoundation.org).
+- **Messari Signals** powered by [Messari](https://messari.io) are AI-generated token reports. For deeper insights, use the  [Messari Deep Research](explore-ai-agents#messari-deep-research) Agent.
+
+![Analyze the perpetual trading market in Warden](../../static/img/warden-app/trade-7.png)
+![See perps on a live chart in Warden](../../static/img/warden-app/trade-8.png)
+
+## Perpetual trading
+
+#### Trade perp
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)